lib/impl/editing.dart

```diff
@@ -101,6 +96,22 @@ bool _handleEnterKey(TextEditor editor, int row, int col) {
       });
 
       return true;
+    } else if (atEol) {
+      var prefLineLength = atom.config
+          .getValue('editor.preferredLineLength', scope: editor.getRootScopeDescriptor());
+
+      if (col >= prefLineLength) {
+        int wrapAtCol = line.substring(0, prefLineLength + 1).lastIndexOf(' ');
```

Comment:
Can you be a bit defensive here? If `wrapAtCol` == -1, we should not wrap, but still continue the comment.